index.js

```diff
@@ -218,7 +218,7 @@ exports.writeMainView = function (dir, cb) {
 exports.writeIcon = function (dir, cb) {
   var iconPath = path.join(__dirname, 'assets/icon.png')
```

Comment:
@toddself good catch! I think `require` statements do the right thing but `path` strings may not -- I always err on the side of not using `/` at all when joining & resolving with `path`.